tests/integration/security/security-response-mapper/src/test/java/io/helidon/tests/integration/security/mapper/RestrictedResourceTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.tests.integration.security.mapper;
+
+import javax.enterprise.inject.se.SeContainer;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+import io.helidon.microprofile.cdi.HelidonContainer;
+import io.helidon.microprofile.server.ServerCdiExtension;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class RestrictedResourceTest {
```

Comment:
If you use `@HelidonTest`, you do not need beans.xml (you can add required beans using `@AddBean`), you also do not need to manually start/stop the container and create a web target (`WebTarget` can be injected into constructor, with the URI of the server prepared)

Reply:
Copy/paste from the other test. Will fix.